src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,34 +1,90 @@
-import { useState } from 'react';
+import { FC, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
+import { getFormInputs } from '../../helpers/form';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
 
-  return (
-    <form className="NewMovie" key={count}>
-      <h2 className="title">Add a movie</h2>
+export const NewMovie: FC<Props> = props => {
+  const { onAdd } = props;
+
+  const [formKey, setFormKey] = useState<number>(0);
+
+  const [titleValue, setTitleValue] = useState<string>('');
+  const [descriptionValue, setDescriptionValue] = useState<string>('');
+  const [imgUrlValue, setImgUrlValue] = useState<string>('');
+  const [imdbUrlValue, setImdbUrlValue] = useState<string>('');
+  const [imdbIdValue, setImdbIdValue] = useState<string>('');
```

Comment:
```suggestion
  const [title, setTitle] = useState<string>('');
  const [description, setDescription] = useState<string>('');
  const [imgUrl, setImgUrl] = useState<string>('');
  const [imdbUrl, setImdbUrl] = useState<string>('');
  const [imdbId, setImdbId] = useState<string>('');
```

But even better would be to create 1 state for all fields, but for now let leave it as it is